[2023-09-05T00:47:57.158+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-05T00:44:55.260775+00:00/task_id=create_listings_raw_table permission to 509
[2023-09-05T00:47:57.159+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-05T00:44:55.260775+00:00/task_id=create_listings_raw_table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-05T00:44:55.260775+00:00/task_id=create_listings_raw_table'
[2023-09-05T00:47:57.187+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-05T00:44:55.260775+00:00/task_id=create_listings_raw_table permission to 509
[2023-09-05T00:47:57.188+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-05T00:44:55.260775+00:00/task_id=create_listings_raw_table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-05T00:44:55.260775+00:00/task_id=create_listings_raw_table'
[2023-09-05T00:47:57.228+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Autotrader_ETL_DAG.create_listings_raw_table manual__2023-09-05T00:44:55.260775+00:00 [queued]>
[2023-09-05T00:47:57.241+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Autotrader_ETL_DAG.create_listings_raw_table manual__2023-09-05T00:44:55.260775+00:00 [queued]>
[2023-09-05T00:47:57.242+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-09-05T00:47:57.263+0000] {taskinstance.py:1382} INFO - Executing <Task(BigQueryCreateEmptyTableOperator): create_listings_raw_table> on 2023-09-05 00:44:55.260775+00:00
[2023-09-05T00:47:57.272+0000] {standard_task_runner.py:57} INFO - Started process 450 to run task
[2023-09-05T00:47:57.276+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Autotrader_ETL_DAG', 'create_listings_raw_table', 'manual__2023-09-05T00:44:55.260775+00:00', '--job-id', '395', '--raw', '--subdir', 'DAGS_FOLDER/autotrader_etl.py', '--cfg-path', '/tmp/tmp9hlcfk_p']
[2023-09-05T00:47:57.279+0000] {standard_task_runner.py:85} INFO - Job 395: Subtask create_listings_raw_table
[2023-09-05T00:47:57.343+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-05T00:44:55.260775+00:00/task_id=create_listings_raw_table permission to 509
[2023-09-05T00:47:57.344+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-05T00:44:55.260775+00:00/task_id=create_listings_raw_table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=Autotrader_ETL_DAG/run_id=manual__2023-09-05T00:44:55.260775+00:00/task_id=create_listings_raw_table'
[2023-09-05T00:47:57.346+0000] {task_command.py:415} INFO - Running <TaskInstance: Autotrader_ETL_DAG.create_listings_raw_table manual__2023-09-05T00:44:55.260775+00:00 [running]> on host 412bb32711f9
[2023-09-05T00:47:57.447+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Autotrader_ETL_DAG' AIRFLOW_CTX_TASK_ID='create_listings_raw_table' AIRFLOW_CTX_EXECUTION_DATE='2023-09-05T00:44:55.260775+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-05T00:44:55.260775+00:00'
[2023-09-05T00:47:57.459+0000] {connection.py:232} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2023-09-05T00:47:57.463+0000] {base.py:73} INFO - Using connection ID 'google_cloud' for task execution.
[2023-09-05T00:47:57.464+0000] {bigquery.py:1507} INFO - Creating table
[2023-09-05T00:47:58.562+0000] {bigquery.py:1535} INFO - Table autotrader_staging.listings_raw already exists.
[2023-09-05T00:47:58.597+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Autotrader_ETL_DAG, task_id=create_listings_raw_table, execution_date=20230905T004455, start_date=20230905T004757, end_date=20230905T004758
[2023-09-05T00:47:58.655+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-05T00:47:58.697+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
